lib/stdlib/test/io_SUITE.erl

```diff
@@ -1043,6 +1044,90 @@ g_choice() ->
     lists:foreach(fun(V) -> g_t(V) end, L),
     ok.
 
+g_ryu() ->
+    %% specific white box tests that should trigger specific edge cases
+    %% to the ryu algorithm
+
+    %% this list is regression tests from the ryu C ref implementation
```

Comment:
I think you can add "https://github.com/ulfjack/ryu/blob/master/ryu/tests/d2s_test.cc" to this comment so that it is easy to find where the test cases re coming from.